packages/backend/src/services/message.ts

```diff
@@ -21,12 +24,11 @@ export const getAllMessages = async (
  * @returns new message document
  */
 export const storeMessage = async (
-  creator: string,
-  content: string
+  params: StoreMessageParameter
 ): Promise<MessageDoc> => {
   const data: MessageInterface = {
-    creator,
-    content,
+    creator: params.creator,
+    content: params.content,
```

Comment:
```suggestion
    creator: creator,
    content: content,
```

Reply:
actually I'm fine using `FanartInterface` as parameter in `storeFanart`, I created the `StoreFanartParameter` just for make sure the two things is a different

Reply:
also because there is a property in `FanartInterface` which is not used as a parameter in `storeFanart`

Reply:
Hmm that might be fair indeed. I don't think it would be safe to make `isVerified in `FanartInterface` optional too. I think for now we can keep using `StoreFanartParameter` then.

Reply:
Oh btw you replied in the wrong thread lmao. Suggestion in this thread is still relevant

Reply:
I deleted my previous reply on this since I took a look at the code, one sec

Reply:
Nevermind. For now I think we can keep using `StoreFanartParameter`

Reply:
ok